test/s-token/s-token-manager.ts

```diff
@@ -286,6 +328,24 @@ contract('STokensManager', ([deployer, user]) => {
 				])
 				expect(_tokenId2).to.equal('2')
 			})
+			it('gives priority to payload based descriptor', async () => {
+				await dev.sTokensManager.setTokenURIDescriptor(
+					property.address,
+					descriptor.address,
+					[web3.utils.keccak256('PAYLOAD')],
+					{ from: user }
+				)
+				await (dev.sTokensManager as any).methods[
+					'setTokenURIDescriptor(address,address)'
+				](property.address, descriptorCopy.address, { from: user })
+				await descriptorCopy.__shouldBe(false)
+				// @ts-ignore
+				await dev.lockup.depositToProperty(
+					property.address,
+					'10000',
+					web3.utils.keccak256('PAYLOAD')
```

Comment:
is this test enough? should it add a test to assert that the payload-based descriptor works correct?